service/src/main/java/eu/europa/ec/fisheries/ers/service/bean/ActivityRulesModuleServiceBean.java

```diff
@@ -107,6 +128,34 @@ public void composeAndSendTripUpdateFaQueryToRules(String tripId) throws Activit
         }
     }
 
+    @Override
+    public void composeAndSendVesselUpdateFaQueryToRules(CreateAndSendFAQueryRequest request) throws ActivityModuleException {
```

Comment:
Try to write tests. It is going to be tough, but take a look at `RulesMovementProcessorBeanTest`. Let's discuss.